DEVELOPMENT.md

```diff
@@ -117,6 +117,13 @@ _note: you may need to add `$HOME/.cargo/bin` to your path in `.env/bin/activate
 ## How to run the tests
 _TODO: Need more thorough tests!_
 
+In order to run the tests, you need bitcoind and elementsd binaries available. For Linux/Mac, there is some support for installing/compiling them. So you can:
+* `./tests/install_bitcoind.sh --bitcoin binary` will install bitcoind in tests/bitcoin
```

Comment:
One needs to have boost installed, otherwise compilation will fail. 
Please add in the documentation
`sudo apt install libboost1.67-all-dev `

Reply:
The script automatically checks (and installs) the missing packages. 
https://github.com/cryptoadvance/specter-desktop/blob/master/tests/install_noded.sh#L136
However, it's also assuming to run as root (line 143).
At least it breaks with a reasonable error-message. The package needed here is imho `libboost-dev` which depends on `libboost1.71-dev` (on my ubuntu 20.04.02 LTS).
Do you agree?

Reply:
Yes I agree. 
This PR seems great.